Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml

```diff
@@ -266,3 +266,263 @@
         hard: false
         layer:
         - LowImpassable
+
```

Comment:
All of these should share a common parent to reduce verbosity. Introduce a base parent like BaseDehydratedXeno, move all common components from these items to that base, and make those items inherit from the new base item (you can make a prototype inherit from multiple parents by supplying a list to the `parent` property, e.g.: `parent: [BaseDehydratedXeno, PlushieFRouny]`. Do note that components from the first parents are inherited first, so if multiple parents have the same component, the resulting prototype will inherit it from the first parent only).

You could also do the same to other rehydratables - I don't know why wizden didn't introduce a common parent for them.

Reply:
the stuff in question has been moved to something in another folder, I don't want to change basegame files though as that's not my pull request.